apis/beacon/blocks/attestations.yaml

```diff
@@ -27,7 +30,9 @@ get:
               data:
```

Comment:
json response also needs the version

Reply:
wdym here? the version in in the header

Reply:
All existing apis that have the version in the header also return it as part of json
https://github.com/ethereum/beacon-APIs/blob/fb718b65d39b62ffd4ee6c95d1d6fb1458c08958/apis/beacon/blocks/block.v2.yaml#L27

Reply:
I see, added the version as a property too. Thanks